jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/rabbitmq/RabbitMqDocumentOutput.java

```diff
@@ -1,6 +1,7 @@
 package ca.bc.gov.open.jrccaccess.autoconfigure.plugins.rabbitmq;
 
 import ca.bc.gov.open.jrccaccess.autoconfigure.AccessProperties;
+import ca.bc.gov.open.jrccaccess.autoconfigure.redis.RedisStorageService;
```

Comment:
question: why do we need to import this into a file where it is not being used?

Reply:
you can configure intellij to remove import on the fly using [optimizing import](https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#optimize-imports), I used to left a lot of those before that ;-) ;-)

Reply:
they are being used.